docs/source/release_notes/release-notes.rst

```diff
@@ -61,7 +61,7 @@ v0.31.0
          - Change
 
     *    - |improved| |devbreak|
-         - Improved performance by preventing excessive reads from the _namespace table when initializing SweepStrategyManager.
+         - Improved Oracle performance on DBKVS by preventing excessive reads from the _namespace table when initializing SweepStrategyManager.
```

Comment:
Do we have a standardised way of referring to DBKVS in the release notes/docs? Is this it?